<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LayerProperties</class>
 <widget class="QWidget" name="LayerProperties">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>317</width>
    <height>294</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Layer Properties</string>
  </property>
  <layout class="QFormLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="titleLabel">
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="title"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="opacityLabel">
     <property name="text">
      <string>Opacity:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="KisSliderSpinBox" name="opacitySlider">
     <property name="suffix">
      <string>%</string>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="blendModeLabel">
     <property name="text">
      <string>Blending Mode:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QComboBox" name="blendMode"/>
   </item>
   <item row="6" column="0">
    <widget class="QLabel">
     <property name="text">
      <string>Visibility:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QCheckBox" name="visible">
     <property name="text">
      <string>Visible to you</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QCheckBox" name="censored">
     <property name="text">
      <string>Censor content</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>5</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="9" column="0">
    <widget class="QLabel">
     <property name="text">
      <string>Collaboration:</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QCheckBox" name="defaultLayer">
     <property name="text">
      <string>Default for new users</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="createdByLabel">
     <property name="text">
      <string>Created by:</string>
     </property>
    </widget>
   </item>
   <item row="11" column="1">
    <widget class="QLabel" name="createdBy">
     <property name="text">
      <string notr="true">#0</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
    </widget>
   </item>
   <item row="13" column="0">
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>5</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="14" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KisSliderSpinBox</class>
   <extends>QSpinBox</extends>
   <header>desktop/widgets/kis_slider_spin_box.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>title</tabstop>
  <tabstop>opacitySlider</tabstop>
  <tabstop>blendMode</tabstop>
  <tabstop>visible</tabstop>
  <tabstop>censored</tabstop>
  <tabstop>defaultLayer</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
